make error (OpenBSD 4.7)

eggdrop git version as of 20181017

```
$ make
[...]
gcc -g -O2 -pipe -Wall -I.. -I..  -DHAVE_CONFIG_H -I/usr/local/include/tcl8.5 -g3 -DDEBUG -DDEBUG_ASSERT -DDEBUG_MEM -DDEBUG_DNS  -c tclhash.c
In file included from tclhash.c:29:
/usr/include/sys/resource.h:56: error: field `ru_utime' has incomplete type
/usr/include/sys/resource.h:57: error: field `ru_stime' has incomplete type
/usr/include/sys/resource.h:96: error: syntax error before "int32_t"
/usr/include/sys/resource.h:101: error: syntax error before "rlim_t"
/usr/include/sys/resource.h:107: error: syntax error before "fixpt_t"
In file included from tclhash.c:29:
/usr/include/sys/resource.h:120: error: syntax error before "id_t"
/usr/include/sys/resource.h:123: error: syntax error before "id_t"
*** Error code 1

Stop in /home/michael/src/eggdrop/src (line 92 of /usr/share/mk/sys.mk).
*** Error code 1

Stop in /home/michael/src/eggdrop (line 256 of Makefile).
```

--- a/src/tclhash.c
+++ b/src/tclhash.c
@@ -26,6 +26,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 
+#include <sys/time.h>
 #include <sys/resource.h>
 #include "main.h"
 #include "chan.h"

